docs/call-numbers.md

```diff
@@ -0,0 +1,19 @@
+# Call Numbers
+
+Note: This documentation was migrated from https://docs.google.com/document/d/1bHvgfgyUmDXV7ROqEZFaRxJFEYhSSUbeQoQT883GMWg/edit#heading=h.oz6ax25g11u and needs to be updated
+
+## Generating browse lists
+
```

Comment:
ssh to catalog-qa1 for staging or catalog.. for production.

Reply:
navigate to /opt/orangelight/current

Reply:
To fully regenerate all browse lists:
-  on catalog-staging `ssh to a staging box`, cd `/opt/orangelight/current` and run: ... (There are no workers for catalog-staging and the rake task needs to run on the catalog box.)
- for catalog production `ssh to catalog-indexer1` (confirm through the crontab that it's running the call numbers and not other rake tasks), cd `/opt/orangelight/current` and run: ...